code/modules/reagents/reagent_containers/glass/bottle/robot.dm

```diff
@@ -17,20 +21,23 @@
 	//icon_state = "bottle16"
 	reagent = INAPROVALINE
 
-/obj/item/weapon/reagent_containers/glass/bottle/robot/inaprovaline/New()
-	..()
-	reagents.add_reagent(INAPROVALINE, 60)
-	return
-
-
 /obj/item/weapon/reagent_containers/glass/bottle/robot/antitoxin
 	name = "internal anti-toxin bottle"
 	desc = "A small bottle of Anti-toxins. Counters poisons, and repairs damage, a wonder drug."
 	icon = 'icons/obj/chemical.dmi'
 	//icon_state = "bottle17"
 	reagent = ANTI_TOXIN
+	
+/obj/item/weapon/reagent_containers/glass/bottle/robot/water
+	name = "internal water bottle"
```

Comment:
No time for a full review but can you break this over someone's head, permanently damaging it? (Thanks @Dacendeth )

Reply:
you'd have to ask the medborgs that too!